WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2564,7 +2564,7 @@ static GSourceFuncs _handlerIntervention =
                 browser->_ignoreLoadFinishedOnce = true;
                 return;
             }
-            browser->OnLoadFailed();
+            browser->OnLoadFailed(failingURI);
```

Comment:
You need similar change for the case when browser plugin is built with WK APIs (wpe-2.22 or older)